wporg-gp-translation-events.php

```diff
@@ -106,6 +106,7 @@ public function gp_init() {
 		GP::$router->add( "/events/$slug/translations/$locale", array( 'Wporg\TranslationEvents\Routes\Event\Translations_Route', 'handle' ) );
 		GP::$router->add( "/events/$slug", array( 'Wporg\TranslationEvents\Routes\Event\Details_Route', 'handle' ) );
 		GP::$router->add( "/events/$slug/attendees", array( 'Wporg\TranslationEvents\Routes\Attendee\List_Route', 'handle' ) );
+		GP::$router->add( "/events/$slug/attendees/remove/$id", array( 'Wporg\TranslationEvents\Routes\Attendee\Remove_Attendee_Route', 'handle' ) );
```

Comment:
The pattern I feel we have been following is to use the slug for pages (i.e. when the URL is visible to the user), and the event id for "api" calls, where the user does not see the URL. I think it would make sense to use the same pattern here.